README.md

```diff
@@ -82,16 +91,32 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
-
+<sentance> ::= "the" [digit{< adjective >}] < noun > < verb > [< adverb >]
+    
+< adjective > ::=  "lazy"|"smelly"
+    
+< noun > ::= "dog"|"cat"
+    
+< verb > ::= "ate"|"ran"
+    
+< adverb > ::= "slowly"|"noisily"
+    
```

Comment:
# -4

This isn't BNF. It's closer to EBNF, but I can't work out what that "digit" means. I was looking for something like

~~~
  <sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
  <adjectives> ::= "" | <adjectives> <adjective>
  <adjective> ::= "lazy" | "smelly"
  <noun> ::= "dog" | "cat"
  <verb> ::= "ate" | "ran"
  <adverb> ::= "slowly" | "noisily" | ""
~~~